erpnext/www/lms/course.html

```diff
@@ -4,143 +4,138 @@
 {% from "www/lms/macros/card.html" import null_card %}
 
 {% block head_include %}
-	<style>
-		div.card-hero-img {
-			height: 220px;
-			background-size: cover;
-			background-repeat: no-repeat;
-			background-position: center;
-			background-color: rgb(250, 251, 252);
-		}
-		.card-image-wrapper {
-			display: flex;
-			overflow: hidden;
-			height: 220px;
-			background-color: rgb(250, 251, 252);
-			justify-content: center;
-		}
-		.image-body {
-			align-self: center;
-			color: #d1d8dd;
-			font-size: 24px;
-			font-weight: 600;
-			line-height: 1;
-			padding: 20px;
-		}
-		section {
-			padding: 5rem 0 5rem 0;
-		}
-		.course-progress {
-			margin-top: 2.5rem;
-		}
-		.course-progress .progress {
-			border-radius: 1rem;
-			border: 1px solid #7ec45e;
-			height: 0.5rem;
-		}
-		.course-progress .progress-bar {
-			background-color: #7ec45e;
-		}
-		.continue-learning-btn {
-			background-color: #76c2ba;
-			color: #fff;
-			border: 2px solid #76c2ba;
-			border-radius: 8px;
-			padding: 10px 30px;
-			text-transform: uppercase;
-			outline: 0;
-		}
-		.continue-learning-btn:hover {
-			background-color: #fff;
-			color: #76c2ba;
-		}
-	</style>
+<style>
+	.image-body {
+		align-self: center;
+		color: #d1d8dd;
```

Comment:
- Use brand primary colour
- Not sticking to the default colour palette will result into bad UX